### PR TITLE
AB#99234 Fixed application list status tag display

### PR DESCRIPTION
**Pull request by MichalSzrederPWC**
### 🛠 Changes being made

Fixed application list status tag display

### 📸 Screenshots (optional)

image

### 🏎 Quality check

- [x] Have you performed local tests?
- [x] Are integration tests passing locally?
- [ ] Have you added some unit tests?
- [ ] Have you added some integration tests?
- [ ] Have you checked WCAG (included screenshot)

### CODE REVIEW

File: src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml.cs [hunk 1]
```diff
@@ -5,8 +5,8 @@ namespace HE.Investments.Common.WWW.Components.ApplicationStatusTagComponent;
 
 public class ApplicationStatusTagComponent : ViewComponent
 {
-    public IViewComponentResult Invoke(ApplicationStatus applicationStatus)
+    public IViewComponentResult Invoke(ApplicationStatus applicationStatus, string additionalClasses)
```
Comment (dawid-pawulski) [inline src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml.cs:8]: string additionalClasses = null

File: src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml [hunk 2]
```diff
@@ -1,3 +1,3 @@
-@model HE.Investments.Common.Contract.ApplicationStatus
+@model (HE.Investments.Common.Contract.ApplicationStatus ApplicationStatus, string AdditionalClasses)
 
-<application-status-tag application-status="@Model"></application-status-tag>
+<application-status-tag application-status="@Model.ApplicationStatus" additional-classes="@Model.AdditionalClasses"></application-status-tag>
```
Comment (dawid-pawulski) [inline src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml:3]: @using HE.Investments.Common.Extensions
@model (HE.Investments.Common.Contract.ApplicationStatus ApplicationStatus, string AdditionalClasses)

<application-status-tag
    application-status="@Model.ApplicationStatus"
    additional-classes="@(Model.AdditionalClasses.IsProvided() ? Model.AdditionalClasses : string.Empty) "></application-status-tag>

File: src/HE.Investment.AHP.WWW/Views/Shared/Components/ApplicationsTable/ApplicationsTable.cshtml.cs [hunk 3]
```diff
@@ -62,6 +62,6 @@ private static DynamicComponentViewModel CreateLinkComponent(ApplicationProjectM
 
     private static DynamicComponentViewModel CreateApplicationStatusComponent(ApplicationProjectModel application)
     {
-        return new DynamicComponentViewModel(nameof(ApplicationStatusTagComponent), new { applicationStatus = application.Status });
+        return new DynamicComponentViewModel(nameof(ApplicationStatusTagComponent), new { applicationStatus = application.Status, additionalClasses = string.Empty });
```
Comment (dawid-pawulski) [inline src/HE.Investment.AHP.WWW/Views/Shared/Components/ApplicationsTable/ApplicationsTable.cshtml.cs:65]: avoid extending method with parameters which are not needed.
Below you have code which might help

File: src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml.cs (hunk 1 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml.cs:8]: done

File: src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml (hunk 2 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Common.WWW/Components/ApplicationStatusTagComponent/ApplicationStatusTagComponent.cshtml:3]: done

File: src/HE.Investment.AHP.WWW/Views/Shared/Components/ApplicationsTable/ApplicationsTable.cshtml.cs (hunk 3 above)
Comment (MichalSzrederPWC) [reply src/HE.Investment.AHP.WWW/Views/Shared/Components/ApplicationsTable/ApplicationsTable.cshtml.cs:65]: done

**Review by dawid-pawulski — APPROVED** [review]

**Review by PawelBargielski — APPROVED** [review]

**Pull request merged**